src/app/components/info-label.tsx

```diff
@@ -0,0 +1,41 @@
+import { ReactNode } from 'react';
```

Comment:
I created this separately, but could likely be merged with the `WarningLabel` and use a `type` prop.